### PR TITLE
Toml configs for OAuthCallBack Handler

### PR DESCRIPTION
**Pull request by wijith7**
https://github.com/wso2/product-is/issues/6449

### CODE REVIEW

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2 [hunk 1]
```diff
@@ -361,9 +361,19 @@
         </UserConsentEnabledGrantTypes>
 
         <OAuthCallbackHandlers>
-        {% for callback_handler in oauth.extensions.callback_handlers %}
-            <OAuthCallbackHandler Class="{{callback_handler}}"/>
-        {% endfor %}
+           {% for callback_handler in oauth.extensions.callback_handlers %}
+              <OAuthCallbackHandler Class="{{callback_handler.class}}"/>
+                {% if priority.value is defined %}
```
Comment (hasinidilanka) [inline features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2]: Change priority.value to callback_handler.priority

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2 [hunk 2]
```diff
@@ -361,9 +361,19 @@
...
+                	<Priority Value="{{priority.value}}"/>
+                {% endif %}
+                {% if property.value is defined %}
```
Comment (hasinidilanka) [inline features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2]: Change property.value to callback_handler.properties

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2 [hunk 3]
```diff
@@ -361,9 +361,19 @@
...
+                  <Properties>
+                    {% for key,value in OAuthCallbackHandler.properties.items() %}
```
Comment (hasinidilanka) [inline features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2]: change OAuthCallbackHandler callback_handler

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2 [hunk 4]
```diff
@@ -361,9 +361,19 @@
         </UserConsentEnabledGrantTypes>
 
         <OAuthCallbackHandlers>
-        {% for callback_handler in oauth.extensions.callback_handlers %}
-            <OAuthCallbackHandler Class="{{callback_handler}}"/>
-        {% endfor %}
+           {% for callback_handler in oauth.extensions.callback_handlers %}
+              <OAuthCallbackHandler Class="{{callback_handler.class}}"/>
+                {% if priority.value is defined %}
+                	<Priority Value="{{priority.value}}"/>
```
Comment (hasinidilanka) [inline features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2]: Change priority.value to callback_handler.priority

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2 [hunk 5]
```diff
@@ -361,9 +361,19 @@
...
+                {% if callback_handler.priority is defined %}
+                	<Priority Value="{{callback_handler.priority}}"/>
+                {% endif %}
+                {% if callback_handler.properties is defined %}
+                  <Properties>
+                    {% for key,value in callback_handler.properties.items() %}
+                      <Properties> name="{{key}}">{{value}} </Properties>
+                    {% endfor %}
+                  </Properties>
+                 {% endif %}
```
Comment (hasinidilanka) [inline features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2]: </OAuthCallbackHandler> tag is missing.

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2 [hunk 6]
```diff
@@ -361,9 +361,19 @@
         </UserConsentEnabledGrantTypes>
 
         <OAuthCallbackHandlers>
-        {% for callback_handler in oauth.extensions.callback_handlers %}
-            <OAuthCallbackHandler Class="{{callback_handler}}"/>
-        {% endfor %}
+           {% for callback_handler in oauth.extensions.callback_handlers %}
+              <OAuthCallbackHandler Class="{{callback_handler.class}}"/>
```
Comment (hasinidilanka) [inline features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2]: Do not end the `OAuthCallbackHandler` tag here 
<OAuthCallbackHandler Class="{{callback_handler.class}}">

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2 (hunk 5 above)
Comment (wijith7) [reply features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2]: fixed

**Pull request merged**